A3A/addons/core/functions/CREATE/fn_createAIAirplane.sqf

```diff
@@ -295,11 +295,12 @@ if (!_busy) then {
 	private _groupX = createGroup _sideX;
 	_groups pushBack _groupX;
 	_countX = 0;
-	while {_countX < 3} do {
+	private _vehCount = round (random [2, 4, 5]);
+	while {_countX < _vehCount} do {
 		private _veh = objNull;
 		private _spawnParameter = [_markerX, "Plane"] call A3A_fnc_findSpawnPosition;
 		if(_spawnParameter isEqualType []) then {
-			private _vehPool = (_faction get "vehiclesPlanesCAS") + (_faction get "vehiclesPlanesAA");
+			private _vehPool = (_faction get "vehiclesPlanesCAS") + (_faction get "vehiclesPlanesAA") + (_faction get "vehiclesPlanesGunship") + (_faction get "uavsAttack");
```

Comment:
same here

Reply:
getOrDefault ["vehiclesPlanesGunship", "vehiclesPlanesCAS"]; and
getOrDefault ["uavsAttack", "vehiclesPlanesCAS"]; 
?